components/ethernet.rst

```diff
@@ -42,10 +54,11 @@ Configuration variables:
   - ``JL1101``
   - ``KSZ8081``
   - ``KSZ8081RNA``
+  - ``W5500``
 
-- **mdc_pin** (**Required**, :ref:`config-pin`): The MDC pin of the board.
+- **mdc_pin** (**Required for RMII chips**, :ref:`config-pin`): The MDC pin of the board.
```

Comment:
We have scripts that are specifically looking for `**Required**` or `*Optional*` in this section.
Perhaps it is better to split the common schema and the RMII and SPI schemas into their own lists? 

```suggestion
- **mdc_pin** (**Required for RMII chips**, :ref:`config-pin`): The MDC pin of the board.
```